src/main/java/com/autotune/utils/kafka/KruizeKafkaConsumer.java

```diff
@@ -0,0 +1,64 @@
+package com.autotune.utils.kafka;
```

Comment:
Please move all of kafka to `autotune/common` instead of `autotune/utils`

Reply:
Done